README.md

```diff
@@ -1,17 +1,39 @@
 # README.md
 
-Hermes is a background process on the Schulich Delta's onboard computer. 
-It rebroadcasts a data feed from the Central Communication System and also logs the data. 
+Hermes is a background process on the Schulich Delta's onboard computer.
+It rebroadcasts a data feed from the Central Communication System and also logs the data.
 
 The name is inspired by the Olympian god Hermes who is portrayed as a messenger and scribe of the gods.
 
+## File Structure
+
+You have now successfully cloned the Hermes git repo, next is ensuring that you have the proper file structure.
+Originally it should look similar to this:
+
+  - "Original Directory"
+    - Hermes
+
+From the directory that this README is currently located in (we assume this is named Hermes), run the command:
+    `cd ../ && mv ./Hermes ./src && mkdir Hermes && mv ./src ./Hermes/`  
+    (replace Hermes with the name of your cloned directory if it is different)
+This will rename the cloned directory to src and move it within a new directory called 'Hermes'
+Your new file structure should look like:
+
+  - "Original Directory"
+    - Hermes
+      - src
+
 ## Testing
 
-Testing the Schulich Delta Desktop Hermes is done with googletest and googlemock, the gmock and gtest header files need to be in your include path /usr/local/include or /usr/include.
-You must compile the library that has the definitions yourself.
-Meaning you need a `libgmock.a` archive file in the test directory in order to compile the tests.
+Testing the Schulich Delta Desktop Hermes is done with googletest and googlemock,
+the gmock and gtest header files need to be in your include path /usr/local/include
+or /usr/include.
```

Comment:
There will be many different widths in use in text editors, let them use their own word wrap, and let one sentence be on one line.